__tests__/player.test.js

```diff
@@ -13,22 +13,22 @@ describe('Player constructor', () => {
 });
 
 describe('Is high score method', () => {
-  it('returns true if empty high scores array', () => {
-    const player = new Player('Clover');
-    const _highScores = [];
-    expect(player.isHighscore()).toBe(true);
-  });
   it('returns true if high scores array length less than 2', () => {
     const player = new Player('Clover');
-    const _highScores = [jest.fn()];
-    _highScores.length = 2
-    expect(player.isHighscore()).toBe(true);
+    player._highScores = [jest.fn()];
+    player._highScores.length = 1
+    expect(player.isHighscore).toBe(true);
```

Comment:
The error message I get here is:
```
Expected value to be:
  true
Received:
  [Function isHighscore]
```
you need to invoke the method (`player.isHighscore()`) - and it should work